.github/workflows/docker-build.yml

```diff
@@ -7,38 +7,40 @@ on:
 jobs:
   docker:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
     steps:
-      -
-        name: Checkout
-        uses: actions/checkout@v2
-      -
-        name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
-      -
-        name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-      -
-        name: Login to DockerHub
-        uses: docker/login-action@v1
+      - name: Set repo name lowercase
+        id: repo
+        uses: ASzc/change-string-case-action@v2
+        with:
+          string: ${{ github.repository }}
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
I'd love an ARM build but do we currently run well on ARM? The pybluemonday dependency (which I admittedly maintain) I'm not sure builds well on ARM.

Reply:
Went through and did a full setup of CTFd. I created some challenges, users, updated pages, and solved challenges with no problem. I used the built in default DB for this test. 
Let me know if you have a specific area of concern that I should test.